dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/EuTaxonomyNonFinancialsFramework.kt

```diff
@@ -0,0 +1,56 @@
+package org.dataland.frameworktoolbox.frameworks.eutaxonomynonfinancials
+
+import org.dataland.frameworktoolbox.frameworks.FrameworkGenerationFeatures
+import org.dataland.frameworktoolbox.frameworks.InDevelopmentPavedRoadFramework
+import org.dataland.frameworktoolbox.intermediate.Framework
+import org.dataland.frameworktoolbox.intermediate.group.ComponentGroup
+import org.dataland.frameworktoolbox.intermediate.group.ComponentGroupApi
+import org.dataland.frameworktoolbox.intermediate.group.edit
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.LabelBadgeColor
+import org.springframework.stereotype.Component
+import java.io.File
+
+/**
+ * Definition of the Eu Taxonomy Non-Financials framework
+ */
+@Component
+class EuTaxonomyNonFinancialsFramework : InDevelopmentPavedRoadFramework(
+    identifier = "eutaxonomy-non-financials",
+    label = "EU Taxonomy Non-Financials",
+    explanation = "Das Eu Taxonomy Non-Financials Framework",
+    File(
+        "./dataland-framework-toolbox/inputs/euTaxonomyNonFinancials/EutaxonomyNonFinancials.xlsx",
+    ),
+    order = 2,
+    enabledFeatures = setOf(
+        FrameworkGenerationFeatures.ViewPage, FrameworkGenerationFeatures.FakeFixtures,
```

Comment:
Warum keine upload page.

Reply:
Weil die mit den aktuellen devtools nicht generierbar ist.
Aktuell kann das devtool nur 3-layered-upload-pages erstellen.

Wenn du aber ins data dictionary schaust siehst du schnell, dass hier manchmal nur zwei layers sind (also eine KPI direkt unter einer top-level-section)

Daher nutzen wir die upload page weiterhin mit der "alten" und statischen uploadconfig.
Das geht v.a. auch deswegen, weil wir das backend-datamodel in der story nicht verändert haben (bis auf eine mini-Umbenennung).

Nachteil ist natürlich, dass man die UploadConfig händisch maintainen muss weiterhin bei Änderungen im data dictionary.

Aber jetzt die Upload-Config-generation anzupassen wäre ein Höllen-Scope-Creep gewesen.